diff --git a/nolearn/dbn.py b/nolearn/dbn.py
index 9deb53f..c5e0636 100644
--- a/nolearn/dbn.py
+++ b/nolearn/dbn.py
@@ -49,7 +49,8 @@ class DBN(BaseEstimator):
 
         verbose=0,
         max_norm=-1,
-        noises=[]
+        noises=[],
+        dropout_adv = 0.0,
         ):
         """
         Many parameters such as `learn_rates`, `dropouts` etc. will
@@ -229,6 +230,7 @@ class DBN(BaseEstimator):
         self.verbose = verbose
         self.max_norm = max_norm
         self.noises = noises
+        self.dropout_adv = dropout_adv
 
     def _fill_missing_layer_sizes(self, X, y):
         layer_sizes = self.layer_sizes
@@ -262,7 +264,8 @@ class DBN(BaseEstimator):
             self.use_re_lu,
             v(self.uniforms),
             max_norm = self.max_norm,
-            noises = self.noises
+            noises = self.noises,
+            dropout_adv = self.dropout_adv
             )
 
         return net
